common/src/main/java/dev/ftb/mods/ftblibrary/sidebar/SidebarButtonCreatedEvent.java

```diff
@@ -1,20 +1,20 @@
 package dev.ftb.mods.ftblibrary.sidebar;
 
-import dev.architectury.annotations.ForgeEvent;
 import dev.architectury.event.Event;
 import dev.architectury.event.EventFactory;
 
 import java.util.function.Consumer;
 
+//Todo move this to api class in the future - unreal
```

Comment:
Please resolve

Reply:
Moving this now would break existing "api" usage https://github.com/FTBTeam/FTB-Quests/blob/bc3d65aaa6072e107caaebd0c57a6276859f8ccf/common/src/main/java/dev/ftb/mods/ftbquests/client/FTBQuestsClientEventHandler.java#L118 More note to move it in future into api so that is so an "api" not random event in random package